app/models/prison.rb

```diff
@@ -49,11 +49,14 @@ def confirm_by(today = Time.zone.today)
   end
 
   def validate_visitor_ages_on(target, field, ages)
+    return if ages.blank?
     adults, _children = ages.partition { |a| adult?(a) }.map(&:length)
-    if adults > MAX_ADULTS
+    # For simplicity’s sake, this operates on the possibly unsafe assumption
+    # that the first visitor is always going to be the requesting visitor.
+    if ages.first < REQUESTING_VISITOR_MIN_AGE
```

Comment:
@tatyree I see what you're saying and happy to go with the lead visitor being the first visit for now. We can always change later if needs be.

Separately, should both checks still be part of the same if-else block – aren't they independent? It's possible for the lead visitor to be a non-adult _and_ for there too many non-child visitors.